### ISSUE TITLE
How to Install cp-ansible 7.0.1 through Ansible Tower

### ISSUE THREAD

**Issue by it4discovery** [confluentinc/cp-ansible#885]
**For Confluent Enterprise Support customers, we would strongly advise you to open a Support ticket which will be addressed within your Support contract SLA at ```https://support.confluent.io```**

**Describe the issue**
How can I install kafka confluent platform, cp-ansible , through Ansible Tower?
So, I have downloaded cp-ansible (7.0.1-post) in our gitlab repo, and I want to install confluent kafka through our Ansible Tower. I have already defined the "project", "inventory" and "template" in our Tower. It stuck in first step as I get: 

ERROR! the role 'variables' was not found in /tmp/awx_30892_7u7yg13d/project/playbooks/roles:/tmp/awx_30892_7u7yg13d/requirements_roles:/var/lib/awx/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/tmp/awx_30892_7u7yg13d/project/playbooks
The error appears to be in '/tmp/awx_30892_7u7yg13d/project/playbooks/all.yml': line 8, column 15, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
    - import_role:
        name: variables

Where and how should I define the variables in Tower?

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**
cp-ansible installs kafka and zookeeper confluent on my servers.

**Inventory File**
Please provide your inventory file in order to aid in the investigation.

**Logs**
Please provide the output of the ```-vvv``` switch from Ansible with the failed log snippet. Please also provide the logs of the specific component which has failed.  If multiple attempts to run have been made, please provide the start and stop time of the component if known.
ERROR! the role 'variables' was not found in /tmp/awx_30892_7u7yg13d/project/playbooks/roles:/tmp/awx_30892_7u7yg13d/requirements_roles:/var/lib/awx/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/tmp/awx_30892_7u7yg13d/project/playbooks
The error appears to be in '/tmp/awx_30892_7u7yg13d/project/playbooks/all.yml': line 8, column 15, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
    - import_role:
        name: variables

**Environment (please complete the following information):**
 - OS: [e.g. RHEL 7.6]
 - CP-Ansible Branch: [5.5.0-post]
 - Ansible Version [e.g. ansible 2.9.6]

**Additional context**

**Comment by erikgb** [comment]
I think installing from Tower might be a challenge, as they are far behind on Ansible version. Please read the docs for 7+ version: https://docs.confluent.io/ansible/current/ansible-download.html#ansible-download

**Comment by it4discovery** [comment]
Yeah, I was able to define variables on template/hosts and run 6.2 through Tower, but the same sort of variables is not working with 7+ !! seems 7+ is quite different !?

**Comment by erikgb** [comment]
Yes, it is a new major version. Based on Ansible Collection. Please read the docs.
